melos.yaml

```
@@ -0,0 +1,46 @@
+command:
+  clean:
+    hooks:
+      pre: melos exec -- "flutter clean"
+
+scripts:
+  # Generate code for all packages using build_runner
+  gen:
+    description: Run build_runner for all packages
+    run: |
+      melos exec --depends-on="build_runner" -- "dart run build_runner build --delete-conflicting-outputs"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)